apps/webapp/app/v3/authenticatedSocketConnection.server.ts

```diff
@@ -50,8 +53,42 @@ export class AuthenticatedSocketConnection {
     ws.addEventListener("close", this.#handleClose.bind(this));
     ws.addEventListener("error", this.#handleError.bind(this));
 
+    ws.on("ping", (data) => {
+      logger.debug("[AuthenticatedSocketConnection] Received ping", {
```

Comment:
I think lets remove these logs, will add a ton of additional events to baselime that is probably not worth it

Reply:
We should never receive any pings, would be good to know if when we do